crates/router/src/configs/settings.rs

```diff
@@ -906,6 +912,11 @@ pub struct ServerTls {
     pub certificate: PathBuf,
 }
 
+#[cfg(feature = "v2")]
+#[derive(Debug, Clone, Deserialize, Default)]
+pub struct CellInformation {
```

Comment:
can you impl default on this struct rather than on the inner field struct?

Reply:
Cannot be done as the internal types are private